src/PokemonGymOwner.java

```diff
@@ -0,0 +1,18 @@
+import java.util.List;
+
+public class PokemonGymOwner extends PokemonTrainer
+{
+    private final String town;
+
+    public PokemonGymOwner(String player, String town,List<Pokemon> pokemons)
```

Comment:
De volgorde van deze parameters is niet helemaal logisch. Beter kun je eerst de super-parameters doen (in de juiste volgorde) en dan de "nieuwe" parameters.